core/accounting.go

```diff
@@ -0,0 +1,72 @@
+package core
+
+import (
+	"math/big"
+	"sync"
+	"time"
+)
+
+// Use transcodeLoopTimeout as TTL for a ManifestID's credit balance
+// We should prevent the use of global state if they are not constants
+
+// Accounts holds credit balances on a per-stream basis
+type Accounts struct {
```

Comment:
Should this be called `Balances`? IMO `Accounts` seems to imply that the struct is will hold ETH account specific data, but in reality the underlying map is just holding balances keyed by a manifestID.

Reply:
I wanted to avoid having `Balances.balances` but we can, yes.

Reply:
Or maybe `BalanceSheet`?

Reply:
I guess Balances is fine considering the underlying map is unexported, so `Balances.balances` would only be internal actually internal to that file. 

What's offputting still is the actually exported getter `Balances.Balance(mid)` , or isn't it ?

Reply:
Either using `BalanceSheet` as the name of the struct or keeping the name `Balances` is fine by me so I'll leave that up to you

Reply:
I will go with Balances for brevity.